return {
	["Show Players"] = {
		Image = 16071029634,
		Stock = 2,
		Price = 150,
		Chance = 70,
        Cooldown = 15,
		Name = "Show Players",
        Description = "Reveal the location of all players.",
		Duration = 5,
    },
	["Bloodlust"] = {
		Image = 16071029634,
		Stock = 1,
		Price = 175,
		Chance = 55,
		Cooldown = 20,
		Name = "Bloodlust",
		Description = "Highlight the closest survivor and deal double damage to them.",
		Duration = 8,
	},
	["Freeze Generators"] = {
		Image = 16071029634,
		Stock = 1,
		Price = 200,
		Chance = 40,
		Cooldown = 20,
		Name = "Freeze Generators",
		Description = "Prevents all generators from being repaired for 15 seconds.",
		Duration = 15,
	},
	["Blackout"] = {
		Image = 16071029634,
		Stock = 5,
		Price = 125,
		Chance = 60,
		Cooldown = 18,
		Name = "Blackout",
		Description = "Reduce all survivors' vision dramatically for 10 seconds.",
		Duration = 10,
	},
	["Hunter's Mark"] = {
		Image = 16071029634,
		Stock = 2,
		Price = 100,
		Chance = 80,
		Cooldown = 10,
		Name = "Hunter's Mark",
		Description = "Mark the nearest survivor, revealing their location for 8 seconds.",
		Duration = 8,
	},
}