return {
    ["Invisibility"] = {
        Image = 133355043444426,
        MaxAmount = 1,
        Price = 175,
        Cooldown = 25,
        Duration = 3,
        Name = "Invisibility",
        Description = "Become invisible for a short duration.",
    },
	["Heal"] = {
		Image = 133355043444426,
		MaxAmount = 2,
		Price = 100,
		Cooldown = 19,
		Name = "Heal",
		Description = "Instantly heal 50 HP.",
		HealAmount = 50,
	},
	["Sprint Burst"] = {
		Image = 114124336527832,
		MaxAmount = 2,
		Price = 150,
		Cooldown = 20,
		Name = "Sprint Burst",
		Description = "Massive speed boost for a short time.",
		Duration = 3,
		SpeedMultiplier = 2.5,
	},
	["Quick Fix"] = {
		Image = 133355043444426,
		MaxAmount = 3,
		Price = 75,
		Cooldown = 15,
		Name = "Quick Fix",
		Description = "Repair generators 100% faster for 10 seconds.",
		Duration = 10,
		RepairBoost = 2,
	},
} 